CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
 ## master
-
+* Adding support for multi-leg routes in `CarPlayManager`. ([#1841](https://github.com/mapbox/mapbox-navigation-ios/pull/1841))
```

Comment:
Since there is a backwards-incompatible public API change, the change needs to be indicated explicitly:

```suggestion
* Renamed `CarPlayManager.calculateRouteAndStart(from:to:completionHandler:)` to `CarPlayManager.previewRoutes(between:completionHandler:)` and added a `CarPlayManager.previewRoutes(for:completionHandler:)` method that accepts an arbitrary `NavigationRouteOptions` object. ([#1841](https://github.com/mapbox/mapbox-navigation-ios/pull/1841))
```